ActindoWorkFlow/UserInterface/01_WorkflowElements.md

```diff
@@ -0,0 +1,203 @@
+[!!Workflows](ActindoWorkFlow)
+
+# Workflow elements
+
+*Workflows > Menu entry Workflows > Tab OVERVIEW > Select workflow > Select workflow version*
+
+![Workflow](/Assets/Screenshots/ActindoWorkFlow/Workflows/Workflow.png "[Workflow]")
+
+A workflow is represented with a number of fixed elements, each of them having specific characteristics. These elements are used to design the workflow, which
+can be edited as necessary according to customer needs. For detailed information, see [Manage workflows](#01_ManageWorkflows.md).
+
+A workflow runs from left to right, from a start point to an end point, which can be in turn linked to another workflow. The data input at the start point, for example a document, goes through the different workflow stages (places and actions) until it reaches the end point. The process is than completed.
+
+> [Info] Data is provided from different system modules and plugins, such as *Invoicing*, *Email*, *OmniChannels*, *Tasks*, among others. The actions available to create workflows depend on the modules installed in the system.
+
+
+## Places
+
+![Place](/Assets/Screenshots/ActindoWorkFlow/Workflows/Place.png "[Place]")
+
+A place is a step in the workflow and is represented by circle, usually with inward and outward arrows.
+
+Places serve as link between actions, joining an output port from the previous action to the corresponding input port of the following action. Places linking mandatory input/output ports are displayed per default and cannot be deleted. Additional places can be added by clicking on the ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button next to the optional input/output port. For detailed information about the types of input/output ports, see [Input and output ports](#input-and-output-port).
+
+When clicking a place, it becomes highlighted and the corresponding element settings are displayed in the settings side bar.
+
+![Settings side bar](/Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar01.png "[Settings side bar]")
+
+The place settings side bar is located to the right of the workspace and displays the following information:
```

Comment:
umformulieren -> in der settings side bar können die einstellungen für den place definiert werden